src/main/content/_includes/head.html

```diff
@@ -26,6 +26,7 @@
   <link rel="manifest" href="/img/favicon/site.webmanifest">
   <link rel="mask-icon" href="/img/favicon/safari-pinned-tab.svg" color="#5bbad5">
   <link rel="shortcut icon" href="/img/favicon/favicon.ico">
+  <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,500,600,700&display=swap" rel="stylesheet">
```

Comment:
we want to keep all resources contained in the app (no external resources) so remove this